### ISSUE TITLE
localhost下报错[WDS] Disconnected!

### ISSUE THREAD

**Issue by helloJincheng** [vuejs/vue-cli#3251]
### Version
3.2.3

### Reproduction link
[http://localhost:8080/](http://localhost:8080/)

### Environment info
```

  System:
    OS: macOS High Sierra 10.13.5
    CPU: (4) x64 Intel(R) Core(TM) i5-5350U CPU @ 1.80GHz
  Binaries:
    Node: 8.11.3 - /usr/local/bin/node
    Yarn: 1.9.4 - /usr/local/bin/yarn
    npm: 6.2.0 - /usr/local/bin/npm
  Browsers:
    Chrome: 71.0.3578.98
    Firefox: 63.0.3
    Safari: 11.1.1
  npmGlobalPackages:
    @vue/cli: 3.2.3
```

### Steps to reproduce
Access to XMLHttpRequest at 'http://192.168.0.100:8080/sockjs-node/info?t=1546613525233' from origin 'http://localhost:8080' has been blocked by CORS policy: The value of the 'Access-Control-Allow-Origin' header in the response must not be the wildcard '*' when the request's credentials mode is 'include'. The credentials mode of requests initiated by the XMLHttpRequest is controlled by the withCredentials attribute.

client?13c5:177 [WDS] Disconnected!

### What is expected?
控制台不报错

### What is actually happening?
我已经使用了最新的 "webpack-dev-server": "^3.1.14"，还是会报错，hmr无法使用

**Comment by CaiPeng1989** [comment]
你需要提供你的代码仓库

**Comment by wuyuweixin** [comment]
这个问题修复过了，你最好提供下重现仓库，方便排查。建议尝试删除yarn.lock或package-lock.json再重新安装依赖试试。

**Comment by sodatea** [comment]
Closing due to inactivity. Please open a new issue with a reference to this one if you can follow up with more information.
